platform-emf/src/main/java/com/softicar/platform/emf/attribute/field/day/EmfDayInput.java

```diff
@@ -37,13 +36,6 @@ public void setValue(Day value) {
 		dayInput.setValue(value);
 	}
 
-	@Override
-	public IDomElement setEnabled(boolean enabled) {
```

Comment:
The method IEmfInput.setEnabled(boolean) is never used, so we don't need to implement it anymore, since IEmfInput no longer extends IDomEnableable or IDomDisableable.